src/AppContainer.js

```diff
@@ -50,9 +52,36 @@ const SetDocumentInfo = ({ documentId, feedUri, onSetDocumentInfo }) => {
     return <div>Setting document info...</div>
 }
 
-const AppContainer = ({ location, initialLoadComplete, children, documentInfo, onSetDocumentInfo, extensionsConfig }) => {
+// Thanks: https://stackoverflow.com/questions/36862334/get-viewport-window-height-in-reactjs
```

Comment:
The app bar is now sticky at the top and scrollbar controls the main content. We need to pass the browser window width/height to the MVSorting view, so we use this custom hook to get the browser window dimensions. This width/height state is changed whenever the browser is resized (very convenient).